src/app/config/no-config/no-config.component.ts

```diff
@@ -10,27 +12,40 @@ import { Config, ConfigService } from '../config.service';
   styleUrls: ['./no-config.component.scss'],
 })
 export class NoConfigComponent implements OnInit {
-  public page = 0;
-  public totalPages = 6;
+  public page: number = 0;
+  public totalPages: number = 7;
```

Comment:
the `:number` is redundant here

Reply:
I like it for consistency, but I can't remove it if it bugs ya.

Reply:
![Screenshot 2020-08-31 at 15 21 43](https://user-images.githubusercontent.com/19818061/91724626-dbe78f00-eb9d-11ea-80ce-a9df63afc605.png)

Actually bugs VSCode :D

Reply:
ahh I use VIM so I don't have that issue, seems like a VSCode issue, but we can remove it since it doesn't hurt anything but my eyes ;)